fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobName.java

```diff
@@ -49,7 +49,8 @@ public enum JobName {
                                                                                                                                                                                                                     "Update Trial Balance Details"), EXECUTE_DIRTY_JOBS(
                                                                                                                                                                                                                             "Execute All Dirty Jobs"), INCREASE_BUSINESS_DATE_BY_1_DAY(
                                                                                                                                                                                                                                     "Increase Business Date by 1 day"), INCREASE_COB_DATE_BY_1_DAY(
-                                                                                                                                                                                                                                            "Increase COB Date by 1 day");
+                                                                                                                                                                                                                                            "Increase COB Date by 1 day"), LOAN_DELINQUENCY_CLASSIFICATION(
+                                                                                                                                                                                                                                                    "Loan Delinquency Classification");
```

Comment:
Let's not add a new job. This PR should be only about the ability to configure delinquency buckets. Job will follow later.

Reply:
Done! removed